Packs/OPSWAT-Filescan/Integrations/OPSWATFilescan/OPSWATFilescan.py

```diff
@@ -0,0 +1,400 @@
+    command_res_ls = []
+    for report in reports:
+        command_res_ls.append(build_one_reputation_result(reports[report]))
+    return command_res_ls
+
+
+def is_valid_pass(api_response: Dict[str, Any]):
+    if "rejected_files" not in api_response:
+        return True
+    for reject in api_response["rejected_files"]:
+        if reject.get("rejected_reason") == "INVALID_PASSWORD":
+            return False
+    return True
+
+
+@polling_function(
+    name=demisto.command(),
+    timeout=arg_to_number(demisto.args().get("timeout", 600)),
+    interval=1,
+    poll_message="Polling result",
+    requires_polling_arg=False,
+)
+def polling_submit_command(args: Dict[str, Any], client: Client):
+
+    if flow_id := args.get("flow_id"):
+        api_response = client.get_scan_result(flow_id)
+        successful_response = False
+
+        if api_response.get("allFinished", False):
+            successful_response = True
+
+        if successful_response:
+            if not is_valid_pass(api_response):
+                raise DemistoException("Invalid password!")
+
+            return PollResult(
+                response=build_reputation_result(api_response), continue_to_poll=False
+            )
+
+        return PollResult(
+            response=[
+                CommandResults(
+                    outputs=api_response,
+                    outputs_prefix=f"{INTEGRATION_CONTEXT_NAME}.Submission",
+                    outputs_key_field="flow_id",
+                    readable_output=f'Waiting for submission "{api_response.get("flow_id")}" to finish...',
+                ),
+            ],
+            continue_to_poll=True,
+            args_for_next_run={"flow_id": args.get("flow_id"), **args},
+        )
+    else:
+        return sample_submission(client, args)
+
+
+""" COMMANDS """
+
+
+def test_module_command(client: Client, *_) -> str:
+    """Performs a basic GET request to check if the API is reachable and authentication is successful."""
+    results = client.test_module()
+    if "accountId" in results:
+        return "ok"
+    raise DemistoException(f"\nTest module failed, {results}")
+
+
+def scan_command(client: Client, args: Dict[str, Any]):
+    return polling_submit_command(args=args, client=client)
+
+
+def search_query_command(client: Client, args: Dict[str, Any]):
+    def validate_args():
+        if page_size and (not page_size.isdigit() or int(page_size) not in [5, 10, 20]):
+            raise DemistoException("Page size value must be 5, 10 or 20")
+        if page and (not page.isdigit() or int(page) <= 0):
+            raise DemistoException("Page must be an integer and grater than 0")
+        if limit and (int(limit) <= 0 or int(limit) > 50):
+            raise DemistoException("Limit must be an integer and between 1 and 50")
+
+    items = []
+    query_string = args.get("query", "")
+    page_size = args.get("page_size")
+    page = args.get("page")
+    limit = args.get("limit", 10)
```

Comment:
```suggestion
    page_size = arg_to_number(args.get("page_size"))
    page = arg_to_number(args.get("page"))
    limit = arg_to_number(args.get("limit", 10))
```

use `arg_to_number` from CommonServerPython

Reply:
Thanks! Fixed.